src/common/strtol.cc

```diff
@@ -43,19 +55,16 @@ long long strict_strtoll(const char *str, int base, std::string *err)
     *err = errStr;
     return 0;
   }
-  if (*endptr != '\0') {
-    errStr = "The option value '";
-    errStr.append(str);
-    errStr.append("'");
-    errStr.append(" contains invalid digits");
-    *err =  errStr;
-    return 0;
-  }
   *err = "";
   return ret;
 }
 
-int strict_strtol(const char *str, int base, std::string *err)
+long long strict_strtoll(const char *str, int base, std::string *err)
+{
+  return strict_strtoll(std::string_view(str), base, err);
```

Comment:
Sweet! And, std::string will also convert to std::string_view, so you should be covered!